tools/Az.Tools.Predictor/Az.Tools.Predictor/AzPredictorService.cs

```diff
@@ -34,6 +34,9 @@ internal class AzPredictorService : IAzPredictorService, IDisposable
     {
         private const string ClientType = "AzurePowerShell";
 
+        // Default to the latest model version?
+        private static readonly Version DefaultModelVersion = new Version(5, 1);
```

Comment:
I wonder if we can assume "5.1" is the latest version. Should we use a special value like `Version(0, 0)` to tell the server to return the latest? That'll need to change at the server side. 
The issue with "5.1" is that if the user doesn't update the module, and the Az module is updated, then it'll not requesting for the latest.

Reply:
That's a great idea. I think it makes sense to give the server the responsibility of knowing what the latest version of the model is and not the client. I'll change this to "0.0" and add the ability to chose the latest based on this value from the server side.